amaranth-chrome-ext/src/AmaranthUtil.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Utilities for the Amaranth Chrome extension.
+ * This class is a wrapper for pure helper functions, therefore all
+ * functions should be static and should contain no internal state.
+ */
+class AmaranthUtil {
+  /** @public @const @type {string} */
+  static get specialCharacters() {
+    return '!"#$%&()*+,-./:;<=>?@[\\]^_`{|}~\t\n';
+  }
+
+  /**
+   * Removes special characters from a string. The characters to remove are
+   * defined in `AmaranthUtil.specialCharacters`.
+   * @param {string} str String to remove special characters from
+   * @return {string} A new string with all characters in `str` except those
+   * present in `AmaranthUtil.specialCharacters`
```

Comment:
s/those present/characters defined in

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/a0a1070e1663d24d48d40ddb484135cc32529d60